src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.module.css

```diff
@@ -0,0 +1,63 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 30px 1fr 30px;
+    grid-template-rows: auto;
+  }
+}
+
+.carouselWrapper {
+  grid-column: 2;
+}
+
+@media (--tablet-and-down) {
+  .carouselWrapper {
+    column-gap: 10px;
+    grid-column: 1/ span 13;
+    grid-column: 1 / span 13;
+    grid-template-columns: repeat(13, minmax(0, 1fr));
+    padding: 0;
+  }
+}
+
+.slideQuote {
+  background: var(--color-gray-300);
+  /*Height 100% is to make all slides the same height */
+  height: 100%;
+  margin: 0;
+  padding: 60px 50px;
+}
+
+.quote {
+  font: var(--font-body-medium);
+  margin: 0;
+}
+
+.attribution {
+  font: var(--font-caption);
+  margin-top: 22px;
+}
+
+.dots {
+  padding-top: 19px;
+  text-align: center;
+}
+
+.dots :global(.carousel__dot) {
+  background-color: var(--color-gray-400);
+  border-radius: 50%;
+  border-width: 0;
+  height: 6px;
```

Comment:
Does this do anything? I think the `padding` makes up for the height.

Also, it looks like these dots are twice as big as the ones in the design, but at the same time I think if we make them any smaller, they become too difficult to click, so maybe it's fine.

Reply:
👍 You're right, it doesn't do anything. Deleted.